rocky/rocky/settings.py

```diff
@@ -4,102 +4,98 @@
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv("SECRET_KEY")
+SECRET_KEY = env("SECRET_KEY")
 
-QUEUE_NAME_BOEFJES = os.getenv("QUEUE_NAME_BOEFJES")
-QUEUE_NAME_NORMALIZERS = os.getenv("QUEUE_NAME_NORMALIZERS")
-QUEUE_URI = os.getenv("QUEUE_URI")
+QUEUE_NAME_BOEFJES = env("QUEUE_NAME_BOEFJES", default="boefjes")
+QUEUE_NAME_NORMALIZERS = env("QUEUE_NAME_NORMALIZERS", default="normalizers")
+QUEUE_URI = env.url("QUEUE_URI", "").geturl()
 
-OCTOPOES_API = os.getenv("OCTOPOES_API")
+OCTOPOES_API = env.url("OCTOPOES_API", "").geturl()
 
-SCHEDULER_API = os.getenv("SCHEDULER_API", "")
+SCHEDULER_API = env.url("SCHEDULER_API", "").geturl()
 
-KATALOGUS_API = os.getenv("KATALOGUS_API", "")
+KATALOGUS_API = env.url("KATALOGUS_API", "").geturl()
 
-BYTES_API = os.getenv("BYTES_API", "")
-BYTES_USERNAME = os.getenv("BYTES_USERNAME", "")
-BYTES_PASSWORD = os.getenv("BYTES_PASSWORD", "")
+BYTES_API = env.url("BYTES_API", "").geturl()
+BYTES_USERNAME = env("BYTES_USERNAME", default="")
+BYTES_PASSWORD = env("BYTES_PASSWORD", default="")
 
-KEIKO_API = os.getenv("KEIKO_API", "")
+KEIKO_API = env.url("KEIKO_API", "").geturl()
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv("DEBUG") == "True"
+DEBUG = env("DEBUG")
 
 # Make sure this header can never be set by an attacker, see also the security
 # warning at https://docs.djangoproject.com/en/4.2/howto/auth-remote-user/
-REMOTE_USER_HEADER = os.getenv("REMOTE_USER_HEADER")
-REMOTE_USER_FALLBACK = os.getenv("REMOTE_USER_FALLBACK", "False").casefold() != "false"
+REMOTE_USER_HEADER = env("REMOTE_USER_HEADER", default=None)
+REMOTE_USER_FALLBACK = env.bool("REMOTE_USER_FALLBACK", False)
 
 if REMOTE_USER_HEADER:
-    # Optional list of default organizations to add remote users to,
-    # format: space separated list of ORGANIZATION_CODE:GROUP_NAME, e.g. `test:admin test2:redteam`
-    REMOTE_USER_DEFAULT_ORGANIZATIONS = os.getenv("REMOTE_USER_DEFAULT_ORGANIZATIONS", "").split()
     AUTHENTICATION_BACKENDS = [
-        "rocky.auth.remote_user.RemoteUserBackend",
+        "django.contrib.auth.backends.RemoteUserBackend",
     ]
     if REMOTE_USER_FALLBACK:
         AUTHENTICATION_BACKENDS += [
             "django.contrib.auth.backends.ModelBackend",
         ]
 
-
 # SECURITY WARNING: enable two factor authentication in production!
-TWOFACTOR_ENABLED = os.getenv("TWOFACTOR_ENABLED", "False" if REMOTE_USER_HEADER else "True").casefold() != "false"
+TWOFACTOR_ENABLED = env.bool("TWOFACTOR_ENABLED", not REMOTE_USER_HEADER)
 
 # A list of strings representing the host/domain names that this Django site can serve.
 # https://docs.djangoproject.com/en/4.2/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = os.getenv("DJANGO_ALLOWED_HOSTS", "*").split()
-
+ALLOWED_HOSTS = env("DJANGO_ALLOWED_HOSTS", default="*").split()
```

Comment:
Using `env.list` for lists would be better, although that would be backwards incompatible so if we want to use `even.list` we need to add compatibility code to support the old syntax for at least one release.